wordpress-mobile/WordPress-iOS
# ULS: Update Auth to use 2FA view with code validation

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/336
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/344

This updates Auth to use the 2FA view with code validation.

To test:
- **Enable the `unifiedGoogle` feature flag.**
- Select `Continue with Google` from either login or signup.
- Select a Google account that:
  - _Does not_ have a WP password set.
  - _Does_ have WP 2FA enabled.
- Enter an invalid code. Verify an error message is displayed.
- Attempt to enter non-numeric characters. Verify you cannot.
- Enter a valid code. Verify you are logged in.

| ![Simulator Screen Shot - iPhone 11 Pro Max - 2020-07-29 at 17 18 09](https://user-images.githubusercontent.com/1816888/88863281-ad058400-d1bf-11ea-80c8-431c059cc8d8.png) | ![Simulator Screen Shot - iPhone 11 Pro Max - 2020-07-29 at 17 18 16](https://user-images.githubusercontent.com/1816888/88863291-b2fb6500-d1bf-11ea-9aac-a80005c9fbf8.png) | ![Simulator Screen Shot - iPhone 11 Pro Max - 2020-07-29 at 17 18 20](https://user-images.githubusercontent.com/1816888/88863319-c4447180-d1bf-11ea-9cd2-c790794b413e.png) |
|--------|-------|-------|

PR submission checklist:
